src/Configuration/Config.Abstractions/src/IConfiguration.cs

```diff
@@ -18,6 +18,14 @@ public interface IConfiguration
         /// <returns>The configuration value.</returns>
         string this[string key] { get; set; }
 
+        /// <summary>
+        /// Tries to get a configuration value for the specified key.
+        /// </summary>
+        /// <param name="key">The key.</param>
+        /// <param name="value">The value.</param>
+        /// <returns><c>True</c> if a value for the specified key was found, otherwise <c>false</c>.</returns>
+        bool TryGet(string key, out string value);
```

Comment:
How breaking do you think this will be?

Reply:
@davidfowl The interface break? Minimal. I don't think Hao is pushing back on this. The breaking change push back is that previously any null value in the JSON is converted to an empty string, and so shows up in config as an empty string, while after the change it will show up in config as null, like it was in the JSON.